docs/competition/virtual-competitions/competition-session-checklist.md

```diff
@@ -25,6 +25,8 @@ This checklist enumerates the tasks that need to be completed to run a virtual c
 
 ## After the Code Submission Deadline
 
+- Download the team code at the deadline
+- Negotiate with teams and handle any late submissions
```

Comment:
Nit: Negotiations should happen before the submission deadline.

Reply:
Yes starting negotiations should happen ebfore, but concluding them and handling the late submission should happen afterwards, so I'll leave it as is.